ui/apps/platform/src/Containers/Policies/Wizard/Step3/PolicyCriteriaModal.tsx

```diff
@@ -80,10 +141,35 @@ function PolicyCriteriaModal({
 
     function addField() {
         const itemKey = activeItems[0].title;
-        const itemToAdd = descriptors.find((descriptor) => descriptor.shortName === itemKey);
+        const itemToAdd = descriptors.find(
+            (descriptor) => descriptor.shortName === itemKey || descriptor.name === itemKey
+        );
         const newPolicyFieldCard = getEmptyPolicyFieldCard(itemToAdd);
 
         addPolicyFieldCardHandler(newPolicyFieldCard);
+        close();
+    }
+
+    const toolbar = (
+        <Toolbar style={{ padding: 0 }}>
+            <ToolbarContent style={{ padding: 0 }}>
+                <ToolbarItem widths={{ default: '100%' }}>
+                    <TreeViewSearch
+                        onSearch={onSearch}
+                        id="input-search"
+                        name="search-input"
+                        aria-label="Search input example"
```

Comment:
What do you think about?
```suggestion
                        aria-label="Filter policy criteria"
```